src/components/UserForm.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { connect } from "react-redux";
+import "../App.css";
+import { submitUserInfoAction } from '../actions/submitUserInfoAction'
+import { updateUserInfoFormAction } from '../actions/updateUserInfoFormAction'
+
```

Comment:
we need propTypes

Reply:
Good point - I'll fix it for the component, but I also now posted an Issue about it - https://github.com/TiK15Ryyb/etasitsit-frontend/issues/10

Reply:
https://github.com/TiK15Ryyb/etasitsit-frontend/blob/2005f48ca6e5615d22a202fcf195cd6e2b30b6c8/src/components/UserForm.js#L18-L25

https://github.com/TiK15Ryyb/etasitsit-frontend/blob/2005f48ca6e5615d22a202fcf195cd6e2b30b6c8/src/components/UserForm.js#L52-L55

Also fixed in same way for Table and Seat.

Reply:
Ended up reverting this and instead using the once done in devops branch.